core/storage/classifier/gae_models.py

```diff
@@ -174,3 +174,91 @@ def create(
 
         training_job_instance.put()
         return instance_id
+
+
+class ClassifierExplorationMappingModel(base_models.BaseModel):
+    """Model for mapping exploration attributes to a ClassifierDataModel.
+
+    The id of instances of this class has the form
+    {{exp_id}}.{{exp_version}}.{{state_name}}
+    """
+
+    # The exploration_id of the exploration to whose state the model belongs.
+    exp_id = ndb.StringProperty(required=True, indexed=True)
+    # The exploration version at the time the corresponding classifier's
+    # training job was created.
+    exp_version = ndb.IntegerProperty(required=True, indexed=True)
+    # The name of the state to which the model belongs.
+    state_name = ndb.StringProperty(required=True, indexed=True)
+    # The ID of the classifier corresponding to the exploration attributes.
+    classifier_id = ndb.StringProperty(required=True, indexed=True)
+
+    @classmethod
+    def _generate_id(cls, exp_id, exp_version, state_name):
+        """Generates a unique ID for the Classifier Exploration Mapping of the
+        form {{exp_id}}.{{exp_version}}.{{state_name}}
+
+        Args:
+            exp_id: str. ID of the exploration.
+            exp_version: int. The exploration version at the time
+                this training job was created.
+            state_name: str. The name of the state to which the classifier
```

Comment:
@seanlip It is done. No idea why it didn't update here. Its updated in Files changed tab